webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -0,0 +1,93 @@
+.ldap-wizard-wrapper {
+    background-color: rgba(var(--center-channel-color-rgb), 0.04);
+
+    .ldap-wizard-content-wrapper {
+        display: flex;
+        overflow: hidden; // Prevent double scrollbars on the body if content overflows
+        flex-grow: 1;
+        padding-top: 20px;
+    }
+
+    
+    .ldap-wizard-sidebar {
+        position: sticky;
+        top: 0;
+        display: flex;
+        width: 319px;
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 40px 52px;
+        gap: 8px;
+
+        .ldap-wizard-sidebar-header {
+            display: flex;
+            align-items: center;
+            padding: 0 0 8px 6px;
+            color: rgba(var(--center-channel-color-rgb), 0.75);
+            font-family: 'Open Sans', sans-serif;
```

Comment:
Can any of this be removed? There's a bunch of font styling that seems like it could be removed since it's already set for the rest of the app

Reply:
I did another pass on the styles, 

Otherwise I cleaned up the styles -- heh, I'm still used to styled components. Will do this reduction from now on.

Reply:
I choose to blame not having common components for common things like this 😛